src/resources/breaking-changes.md

```diff
@@ -0,0 +1,873 @@
+---
+title: Breaking changes and deprecations
+description: A list of breaking changes by release in Dart.
+---
+
+This page lists all language and library breaking changes and deprecations in Dart,
+organized by release and area, to help Dart users understand and manage their
+impact. Complete release notes are available in the [Dart SDK changelog][changelog].
+The [breaking change policy][] document describes the policy and process
+around breaking changes and deprecations in Dart. 
+
+**This page includes the following types of breaking changes**:
+
+* **Unversioned**: The Dart SDK does not maintain backward compatibility, and
+  code may break as soon as you [upgrade your sdk version][sdk] if it relies on
+  the previous behavior.
+  
+  *These are the majority of changes and are not specially marked in this list.*
```

Comment:
Consider using underscores (`_`) to indicate italic to visually differentiate the raw markdown from bold.